_data/sidebars/home_sidebar.yml

```diff
@@ -82,6 +82,9 @@ entries:
       url: /upgrading/
       output: web
 
+    - title: Integrate Firebase
```

Comment:
Should this title match the title we use on the homepage? (Building cross-platform Firebase app)